jobs/service-fabrik-serviceflow-manager/templates/config/settings.yml.erb

```diff
@@ -0,0 +1,264 @@
+    include_response_body: <%= link("broker").p('monitoring.include_response_body') %>
+    events_logged_in_db: <%= link("broker").p('monitoring.events_logged_in_db') %>
+    unauthorized:
+      http_status: [401,403]
+      event_name: 'unauthorized_access'
+      description: 'Unauthorized access to service attempted'
+      tags: ['security', 'unauthorized']
+      include_response_body: <%= link("broker").p('monitoring.unauthorized.include_response_body') %>
+  riemann:
+    host: <%= link("broker").p('riemann.host') %>
+    port: <%= link("broker").p('riemann.port') %>
+    protocol : tcp
+    show_errors: <%= link("broker").p('riemann.show_errors') %>
+    prefix: <%= link("broker").p('riemann.prefix') %>
+    log_additional_event: <%= link("broker").p('riemann.log_additional_event') %>
+    http_status_codes_to_be_skipped: <% link("broker").p('riemann.http_status_codes_to_be_skipped').each do |code| %>
+    - <%= code %><% end %>
+
+  ###################
+  # QUOTA MANAGEMENT SETTINGS #
+  ###################
+  quota: <%= JSON.dump(link("broker").p('quota', nil)) %>
+
+  ###################
+  # BACKUP SETTINGS #
+  ###################
+  backup:
+    retention_period_in_days: <%= link("broker").p('backup.retention_period_in_days') %>
+    max_num_on_demand_backup: <%= link("broker").p('backup.max_num_on_demand_backup') %>
+    status_check_every: <%= link("broker").p('backup.status_check_every') %>
+    retry_delay_on_error: <%= link("broker").p('backup.retry_delay_on_error') %>
+    lock_check_delay_on_restart: <%= link("broker").p('backup.lock_check_delay_on_restart') %>
+    backup_restore_status_poller_timeout: <%= link("broker").p('backup.backup_restore_status_poller_timeout') %>
+    backup_restore_status_check_every: <%= link("broker").p('backup.backup_restore_status_check_every') %>
+    abort_time_out: <%= link("broker").p('backup.abort_time_out') %>
+    consecutive_backup_failure_sla_count: <%= link("broker").p('backup.consecutive_backup_failure_sla_count') %>
+    num_of_allowed_restores: <%= link("broker").p('backup.num_of_allowed_restores') %>
+    restore_history_days: <%= link("broker").p('backup.restore_history_days') %>
+    reschedule_backup_delay_after_restore: <%= link("broker").p('backup.reschedule_backup_delay_after_restore') %>
+    provider: <%= JSON.dump(link("broker").p('backup.provider', nil)) %>
+
+  #################################
+  # VIRTUAL HOST MANAGER SETTINGS #
+  #################################
+  virtual_host:
+    provider: <%= JSON.dump(link("broker").p('virtual_host.provider', nil)) %>
+
+  #########################
+  # CLOUDFOUNDRY SETTINGS #
+  #########################
+  cf:
+    url: <%= link("broker").p('cf.url') %>
+    username: <%= link("broker").p('cf.username') %>
+    password: <%= link("broker").p('cf.password') %>
+
+  ###################
+  # DOCKER SETTINGS #
+  ###################
+  docker:
```

Comment:
Do we need docker settings in this job ?

Reply:
@sauravmndl  @amitmalav - with regards to settings its a long pending discussion/debate. If we start adding props selectively we will miss-out things. So my idea or recommendation until we fix this setting issue is to have the same copy of settings across all jobs. This way we just update in one place and copy paste the entire same thing across jobs. 
In case this strategy is well defined, please share the guidelines and will be happy to follow it.

Reply:
@amitmalav @sauravmndl @kamath-prasad I have gone ahead and included just the min props required for the job. Going fwd we can set this as the guideline for any job. Also, we can add a backlog to clear existing settings from other jobs.

Reply:
@jagadish-kb : Yes, i have added a internal jira for the same . Will plan to pick it up in of the sprints.